### PR TITLE
Moving the vhd-footer and azureblobupload to upstream repo

### PR DESCRIPTION
**Pull request by gburges**
```release-note
NONE
```

### CODE REVIEW

**Comment by ehashman** [comment]
/retest

Registry flake

File: cmd/vhdfooter/vhdfooter.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		CurrentSize:       *size,
+		DiskGeometry:      0xffff10ff, // 65535 cylinders, 16 heads, 255 sectors
+		DiskType:          2,          // fixed
+		UniqueID:          id,
+	}
+
+	copy(f.Cookie[:], []byte("conectix"))
+	copy(f.CreatorApplication[:], []byte("qem2"))
+	copy(f.CreatorHostOS[:], []byte("Wi2k"))
+
```
Comment (charlesakalugwu) [inline cmd/vhdfooter/vhdfooter.go]: you are missing this chunk
```
err = f.CalculateChecksum()
if err != nil {
	panic(err)
}

err = binary.Write(os.Stdout, binary.BigEndian, f)
if err != nil {
	panic(err)
}
```

**Comment by ehashman** [comment]
/retest

timeout on RG creation

File: cmd/vhdfooter/vhdfooter.go (hunk 1 above)
Comment (gburges) [reply cmd/vhdfooter/vhdfooter.go]: pushed, thanks!

**Pull request merged**